RadixWallet/Features/AccountDetailsFeature/Coordinator/AccountDetails+Reducer.swift

```diff
@@ -277,4 +289,18 @@ public struct AccountDetails: Sendable, FeatureReducer {
 			}
 		}
 	}
+
+	private func scheduleFetchAccountPortfolioTimer(_ state: State) -> Effect<Action> {
+		.run { [address = state.account.address] _ in
+			await withTaskCancellation(id: CancellableId.fetchAccountPortfolio, cancelInFlight: true) {
```

Comment:
Is there any difference between this `withTaskCancellation` and adding `.cancellable(id: CancellableId.fetchAccountPortfolio, cancelInFlight: true)` to the `.run` effect?

Reply:
It does the same thing, and yes, `.cancellable` looks cleaner. Updated